app/views/assessor_interface/professional_standing_requests/edit_request.html.erb

```diff
@@ -0,0 +1,38 @@
+<% title = "Request LoPS verification" %>
```

Comment:
This page is missing a `content_for :page_title` block, for example:

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/2e9a6e6274bf20a845db934a9decc8a680d4a0e0/app/views/assessor_interface/professional_standing_requests/edit_review.html.erb#L3